django/db/models/base.py

```diff
@@ -684,6 +684,10 @@ def _set_pk_val(self, value):
 
     pk = property(_get_pk_val, _set_pk_val)
 
+    @property
+    def pk_str(self):
+        return self._meta.pk.value_to_string(self)
```

Comment:
I know adding this may be contentious but we could use it everywhere else `value_to_string()` is used - in serialisation & sessions.